driver/examples/configs/baroclinic_c12_from_serialbox.yaml

```diff
@@ -4,10 +4,11 @@ stencil_config:
   validate_args: true
   format_source: false
   device_sync: true
-initialization_type: serialbox
-initialization_config:
-  path: /test_data/8.0.0/c12_6ranks_baroclinic_dycore_microphysics/driver
-  serialized_grid: True
+initialization:
+  type: serialbox
+  config:
+    path: /test_data/8.0.0/c12_6ranks_baroclinic_dycore_microphysics/driver
+    serialized_grid: True
```

Comment:
I think it can be either, but it should be `true` so I'll update it to that. I think with True it only works because we're parsing the yaml in python, but I'm not sure.